framework/filters/AccessRule.php

```diff
@@ -90,6 +92,62 @@ class AccessRule extends Component
      */
     public $denyCallback;
 
+    /** 
+     * @var array a map for passing parameters to the [[User::can()]] function
```

Comment:
how about allowing for callbacks? this would ease the implementation a lot.

'params' => function ($action, $accessRule) {
     return ['autor_id' => Yii::$app->user->id];
}